demo/src/main/java/com/eis/geoCalendar/demo/Localization/LocationManager.java

```diff
@@ -0,0 +1,134 @@
+package com.eis.geoCalendar.demo.Localization;
+
+import android.Manifest;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.location.Location;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationAvailability;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.tasks.OnCanceledListener;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+
+import static com.google.android.gms.location.LocationRequest.PRIORITY_HIGH_ACCURACY;
+
+/**
+ * @author Turcato
+ * <p>
+ * TODO: this is here temporarely just for the demo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comment matches to-do format 'TODO:'.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4824932)